frontend/src/components/MenuPage/MenuPage.jsx

```diff
@@ -8,7 +8,7 @@ function MenuPage() {
       </div>
       <div className="menu-container">
         <h1>Menu</h1>
-        <p>Available meals in our place</p>
+        <p className="Available">Available meals in our place</p>
```

Comment:
please keep the naming consistent :) Lets use lowercase letter at the beginning please